ArduCopter/mode.cpp

```diff
@@ -918,30 +918,22 @@ Mode::AltHoldModeState Mode::get_alt_hold_state(float target_climb_rate_cms)
 
 // transform pilot's yaw input into a desired yaw rate
 // returns desired yaw rate in centi-degrees per second
-float Mode::get_pilot_desired_yaw_rate(int16_t stick_angle)
+float Mode::get_pilot_desired_yaw_rate(float yaw_in)
 {
+    // todo: provide normalised commands from RC library so this can be removed.
+    yaw_in /= ROLL_PITCH_YAW_INPUT_MAX;
```

Comment:
should be able to use `channel_yaw->norm_input()`

Reply:
NICE!! I didn't know that got put in there.